fiftyone/utils/sam3.py

```
@@ -545,3 +551,738 @@ def _forward_pass_auto(self, imgs):
+    See :class:`fiftyone.utils.torch.TorchImageModelConfig` for additional
+    arguments.
+
+    Args:
+        operation_mode ("concept"): concept or visual mode of operation for inference
+        propagation_direction ("both"): direction to propagate in video;
+            supported values are ``"forward"``, ``"backward"``, and ``"both"``
+        prompt_frame_indices (None): 1-based frame indices for visual / exemplar prompts
+        text_frame_idx (1): 1-based frame index for text concept prompts
+    """
+
+    def __init__(self, cfg_dict):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)